app/cypress/support/commands.js

```diff
@@ -84,9 +84,13 @@ Cypress.Commands.add(
   "fillAndCheckNewProjectForm",
   (fundingStream, fundingStreamYear) => {
     cy.url().should("include", "/new");
-    cy.findByLabelText(/Funding Stream$/i).select(fundingStream);
-    cy.findByLabelText(/RFP Year/i).select(fundingStreamYear);
-
+    cy.get("[id=select-parent-dropdown-root_fundingStreamRfpId]")
+      // selecting twice ensures cypress selects the correct funding stream
+      .select(fundingStream)
+      .select(fundingStream);
```

Comment:
I'm not able to notice a difference in testing with just one or two of these statements. Interestingly, asserting that the funding stream should be visible after it is selected causes an error `cy.contains(fundingStream).should("be.visible"); // error`